vmdb/app/assets/javascripts/cfme_application.js

```diff
@@ -243,8 +243,12 @@ function miqDimDiv(divname, status) {
 // Check for changes and prompt
 function miqCheckForChanges() {
   if(cfmeAngularApplication.$scope) {
-    if (cfmeAngularApplication.$scope.form.$dirty)
-      return confirm("Abandon changes?");
+    if (cfmeAngularApplication.$scope.form.$dirty) {
+      answer = confirm("Abandon changes?");
+      if(answer == true)
+        cfmeAngularApplication.$scope.form.$setPristine(true);
+      return answer;
+    }
```

Comment:
sounds good...just didn't want you to get too far and then have to undo too much.